### ISSUE TITLE
Next slide preview is crazy small

### ISSUE THREAD

**Issue by dylanratcliffe** [binford2k/showoff#699]
Chrome on OSX, the next slide preview is about 80% smaller than everything else, the zoom level is linked because of chrome so you can't zoom in as it zooms everything else too

**Comment by dylanratcliffe** [comment]
Note, this is when you are using a floating window

**Comment by binford2k** [comment]
Do you mean like this?

![screen shot 2017-02-28 at 5 02 14 pm](https://cloud.githubusercontent.com/assets/1392917/23441535/bf28fa32-fdd7-11e6-96e3-7d14db736504.png)

It was intentionally made small with the idea that you'd know the slides well enough to recognize them by sight. That said, I see now that it's not zooming properly when I resize the window bigger.

**Comment by dylanratcliffe** [comment]
Ah okay, the problem is I'm not sitting at the computer most if the time
and while I can recognize them by sight, not from 3m away. It would be
awesome if they just filled the window as you resize it but I realise
that's hard

**Comment by binford2k** [comment]
Nah. It's supposed to zoom to fill the window when you resize it. It's a bug.

**Comment by marrero984** [comment]
Fails to copy over the JS.
